camel_converter/__init__.py

```diff
@@ -111,16 +111,26 @@ def to_camel(snake_string: str) -> str:
 
 
 @lru_cache(maxsize=4096)
-def to_snake(camel_string: str) -> str:
+def to_snake(camel_string: str, treat_digits_as_capitals: bool = False) -> str:
```

Comment:
Let make this named only. That way if there are more parameters added later it will be easier to keep them from being breaking changes. 

```suggestion
def to_snake(camel_string: str, *, treat_digits_as_capitals: bool = False) -> str:
```

Reply:
Sure, that makes sense